source/sass/patterns/molecules/tag-list.scss

```diff
@@ -14,15 +23,32 @@
 .tag-list__list {
   @include list-style-none();
   @include label-tag-typography();
+  @include margin(0, block-end);
 }
 
 .tag-list__list--single-line {
   @include truncate-with-ellipsis();
+  $_exclusion-zone: 2rem;
+  --inline-size: calc(100vw - #{$_exclusion-zone} - (2 * (var(--GRID-EDGE-SPACE) + #{$contentHeader-downloadIcon-inline_size__default})));
```

Comment:
This shouldn't be here, the Tag List shouldn't know anything about Content Header. Should set a class on the Content Header author list etc, and set the inline size there.

Reply:
Favour renaming this class to `tag-list__list--content-header`. That name is more accurate, and it creates the logical association to the content header. I'm reluctant to reduce encapsulation by moving the sizing of the pattern out of the pattern.

Reply:
What about a wrapper in the Content Header?

Reply:
Trying to avoid letting Tag List know what it's contained in. Feels very weird that it knows if it's next to a download icon. (Related: what happens if there is no icon?)